modules/images-configuration-blocked.adoc

```diff
@@ -61,12 +61,40 @@ The Machine Config Operator (MCO) watches the `image.config.openshift.io/cluster
 ifndef::openshift-rosa,openshift-dedicated[]
 .Verification
 
-* To check that the registries have been added to the policy file, use the following command on a node:
-// cannot create resource "namespaces"
+* To check that the registries have been added to the policy file, enter the following command to obtain a list of your nodes:
 +
 [source,terminal]
 ----
-$ cat /host/etc/containers/registries.conf
+$ oc get node
+----
++
+Example output
++
+[source,terminal]
+----
+NAME                                       STATUS   ROLES                  AGE   VERSION
+ci-ln-yjrspt2-72292-7jqm5-master-0         Ready    control-plane,master   37m   v1.27.8+4fab27b
+----
+
+. Run the following command to enter debug mode on the node:
++
+[source,terminal]
+----
+$ oc debug node/ci-ln-yjrspt2-72292-7jqm5-master-0 
+----
+
+. When prompted, enter `chroot/host` into the terminal:
++
+[source,terminal]
+----
+sh-4.4# chroot/host
+----
+
+. Enter the following command to check that the registries have been added to the policy file:
++
+[source,terminal]
+----
+sh-5.1# cat etc/containers/registries.conf
```

Comment:
There's no JSON file for blocked registries?

Reply:
No; I'm not sure why it's different, but it is :shrug: